manifest/manifest.go

```diff
@@ -74,21 +71,12 @@ type EcConfig struct {
 	CommiteeLookback         uint64
 }
 
-// Manifest identifies the specific configuration for
-// the F3 instance currently running.
+// Manifest identifies the specific configuration for the F3 instance currently running.
 type Manifest struct {
-	// Sequence number of the manifest.
-	// This is used to identify if a new config needs to be applied
-	Sequence uint64
 	// Initial instance to used for the f3 instance
 	InitialInstance uint64
 	// BootstrapEpoch from which the manifest should be applied
 	BootstrapEpoch int64
-	// Flag to determine if the peer should rebootstrap in this configuration
-	// change at BootstrapEpoch
-	ReBootstrap bool
```

Comment:
Removed because there was no safe way to synchronize non-rebootstrapping reconfigs.